skiros2_pyrobosim_lib/pyrobosim_solutions.py

```diff
@@ -35,3 +45,28 @@ def expand(self, skill):
             self.skill("Navigate", "", remap={"TargetLocation": "ObjectTargetLocation"}),
             self.skill("Place", "", remap={"PlacingLocation": "ObjectTargetLocation"})
         )
+
+
+class problem_2_solution(SkillBase):
+    """
+    """
+    def createDescription(self):
+        self.setDescription(Problem2Solution(), self.__class__.__name__)
+
+    def expand(self, skill):
+        skill.setProcessor(SerialStar())
+        skill(
+            self.skill("Navigate", "", remap={"TargetLocation": "ObjectTargetLocation"}),
+            self.skill("BbUnsetParam", "", remap={"Parameter": "StartLocation"}),
+            self.skill(Selector())(
+                self.skill("OpenLocation", "", remap={"OpenableLocation": "ObjectTargetLocation"}),
+                self.skill("Success", ""),
+            ),
```

Comment:
It would be nice to find a neat way around this.  
One can probably work with the different preconditions on an implementation of a skill description